bootstrap-salt.sh

```diff
@@ -3057,11 +3079,14 @@ __install_epel_repository() {
         rpm -q curl > /dev/null 2>&1 || yum -y install curl
         # rpm from CentOS/RHEL release 5 does not support HTTP downloads
         # properly, especially 3XX code redirects
+        # FIXME: cleartext download over unsecure protocol (HTTP)
         __fetch_url /tmp/epel-release.rpm "http://download.fedoraproject.org/pub/epel/5/${EPEL_ARCH}/epel-release-5-4.noarch.rpm" || return 1
```

Comment:
This is not just insecure, but rather ugly and unnecessary these days. I know because I wrote this particular line as quick workaround for CentOS 5 rpm issue :-)
See discussions here: #784 #724